lib/koans/10_processes.ex

```diff
@@ -11,13 +11,32 @@ defmodule Processes do
     assert information[:status] == ___
   end
 
+  koan "Processes are referenced by their process ID (pid)" do
+    assert is_pid(self) == ___
+  end
+
   koan "New processes are spawned functions" do
     pid = spawn(fn -> nil end)
 
     assert Process.alive?(pid) == ___
   end
 
-  koan "You can send messages to processes" do
+  koan "You can kill processes other than yourself" do
+    pid = spawn(fn -> receive do end end)
+
+    assert Process.alive?(pid) == ___
+    Process.exit(pid, :kill)
+    assert Process.alive?(pid) == ___
+  end
+
+  koan "However, trying to exit normally has no effect" do
```

Comment:
I don't have any strong opinion. My thoughts when looking back at these were general "eh, not so awful I suppose". Whatever you'd like 👍